src/util/index.js

```diff
@@ -48,24 +53,29 @@ export function debounce (cbFunc: () => void): Function {
   };
 }
 
-export const empty = <T> (val: T): boolean => val == null;
+export const empty = <T>(val: T): boolean => val == null;
```

Comment:
again any reason why is this generic? I gues `mixed` should be sufficient in this case ?

Reply:
Oh that, yeah, I saw that but didn't change that in this PR. Happy for you to do a fix / push if you want.